setup.py

```diff
@@ -30,16 +30,14 @@
 
 from __future__ import absolute_import
 
-import sys
 import re
 import os
 import platform
-from setuptools import setup, find_packages
 
-version = getattr(sys, "version_info", (0,))
+import ez_setup
+ez_setup.use_setuptools('20.3')
 
```

Comment:
Is it possible to just do "setup_requires=['setuptools>=20.3']" or so instead of including ez_setup...?